test/topology/conftest.py

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+# This file configures pytest for all tests in this directory, and also
+# defines common test fixtures for all of them to use
+
+import pathlib
+import sys
+
+# Add test.pylib to the search path
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[1]))
+
+
+# By default, tests run against a CQL server (Scylla or Cassandra) listening
+# on localhost:9042. Add the --host and --port options to allow overiding
+# these defaults.
+def pytest_addoption(parser):
+    parser.addoption('--host', action='store', default='localhost',
+                     help='CQL server host to connect to')
+    parser.addoption('--port', action='store', default='9042',
+                     help='CQL server port to connect to')
+    parser.addoption('--ssl', action='store_true',
+                     help='Connect to CQL via an encrypted TLSv1.2 connection')
```

Comment:
A previous review asked why does this parser.option() is in test/topology/conftest.py, but the use of it is in test/conftest.py. What was the conclusion?

Reply:
I moved it. It's a weird thing of github pull requests is that  you begin reviewing before I finish, but as soon as I force-push. And I need to be force-pushing to save work.

Reply:
You can force-push to another branch to save work, and only force-push to the PR-d branch after you finished with everything.